web/frontend/src/lib/components/robot-client.svelte

```diff
@@ -165,8 +182,42 @@ const loadCurrentOps = async () => {
   const list = await getOperations($robotClient);
   const ops = [];
 
+  // $rtt is, from what I can tell, a mediocre measurement of RTT. Looks like
+  // every time we `getOperations` from the robot, we time it and reset $rtt.
+  // I would rather rely on RTT measurements returned by `getStats` on the peer
+  // connection (see below).
   $rtt = Math.max(Date.now() - now, 0);
 
+  // Here might be a good place to gather stats. Gathering immediately after
+  // connection establishment will likely show little to no RTT information, as
+  // the new connection hasn't been used yet.
+  if (!gotStats) {
```

Comment:
Are stats collected routinely or is this only ran once?

Reply:
Only run once per connection; see my comment above the `gotStats` declaration.